free-2.md

```diff
@@ -58,12 +117,37 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+This project would probably have less than half of the time spent working on
+it dedicated to the language design, since we would rest heavily upon the
+host language for the parsing of expressions and the design decisions
+I would be responsible for would mainly be how a user would represent information
+about their table and the method by which they will fill it in.
+
+A good bit of time would likely be spent implementing the process by which a
+recursive expression is transformed into a rule for filling out one cell based
+on others.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This project seems to be pretty modest in scope.  It would involve parsing,
+abstracting the generalized notion of a dynamic programming solution, and then
```

Comment:
Hmmm I would have thought that it was rather ambitious to be able to programmatically turn any recursive function into a DP function.  I know I've had difficulty making decisions such as how to fill in the DP table (left to right, diagonally, etc.).  Perhaps you have a better understanding of DP than me so it could be less ambitious for you...